python/bindings/sidekiq_tx_python.cc

```diff
@@ -37,6 +37,9 @@ void bind_sidekiq_tx(py::module& m)
         std::shared_ptr<sidekiq_tx>>(m, "sidekiq_tx", D(sidekiq_tx))
 
         .def(py::init(&sidekiq_tx::make),
+           py::arg("_card"),
+           py::arg("port_id"),
+           py::arg("port_id2"),
```

Comment:
is this always going to be `100` or `None`?  If so should it be removed?

Reply:
This should be fixed, but this is generated code.